clients/cobol-lsp-vscode-extension/src/__tests__/MiddlewareTest.ts

```diff
@@ -28,20 +28,33 @@ describe("Copybook downloader", () => {
         const copybookResolverURI: any = {
             resolveCopybookURI: jest.fn().mockResolvedValue("copybookUri"),
         };
-        const middleware = new Middleware(null, copybookResolverURI);
-        const params = constructParams("broadcom-cobol-lsp.copybook.cobFile.bookName");
+        const middleware = new Middleware(copybookResolverURI, null);
+        const params = constructParams("broadcom-cobol-lsp.copybook-resolve.cobFile.bookName");
         expect(await middleware.handleConfigurationRequest(params, null, null)).toEqual(["copybookUri"]);
         expect(copybookResolverURI.resolveCopybookURI).toHaveBeenCalledWith("bookName", "cobFile");
     });
     it("Handle copybook request for name with dots", async () => {
         const copybookResolverURI: any = {
             resolveCopybookURI: jest.fn().mockResolvedValue("copybookUri"),
         };
-        const middleware = new Middleware(null, copybookResolverURI);
-        const params = constructParams("broadcom-cobol-lsp.copybook.USER.CLIST.COB.bookName");
+        const middleware = new Middleware(copybookResolverURI, null);
```

Comment:
can we extract this middleware, params, etc. in desc. with let to reuse

Reply:
It could be even a const. Good point.